python_modules/dagster/dagster_tests/daemon_tests/test_queued_run_coordinator_daemon.py

```diff
@@ -140,6 +140,33 @@ def test_get_queued_runs_max_runs(num_in_progress_runs, workspace, daemon):
         assert len(instance.run_launcher.queue()) == max(0, max_runs - num_in_progress_runs)
 
 
+def test_disable_max_concurrent_runs_limit(workspace, daemon):
+    with instance_for_queued_run_coordinator(max_concurrent_runs=-1) as instance:
+        # create ongoing runs
+        in_progress_run_ids = ["in_progress-run-{}".format(i) for i in range(5)]
```

Comment:
should more than 10 in progress runs be started to ensure setting to -1 disables the default limit?

Reply:
Changed to 6 runs below so it would go over the default limit